troposphere/static/js/public_site/bootstrapper.react.js

```diff
@@ -1,24 +1,13 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import Profile from "models/Profile";
 import $ from "jquery";
+
 import Router from "../Router";
+import Profile from "models/Profile";
 import routes from "./AppRoutes.react";
-
-
 import modals from "modals";
-
-modals.PublicModals = require("modals/PublicModals");
```

Comment:
`PublicModals` was no included in the `inlining` of `modals`.  These are modals that may be used in a public/non-authenticated scenario. I do not see where it is addressed. I think that is an oversight.